lib/xgu/xgu.c

```diff
@@ -0,0 +1,29 @@
+#include "xgu.h"
+
+#include <assert.h>
+#include <pbkit/pbkit.h>
+
+DWORD *pushbuffer_edge;
+
+static DWORD* push_command(DWORD command, int nparams) {
```

Comment:
Continue using `DWORD* p` for now, so you can always search & replace for the first argument (it's a hack). We can later switch the API design to remove that argument if we decide on it